testsuite/tests/kuadrant/authorino/metrics/conftest.py

```diff
@@ -16,9 +19,18 @@ def prometheus(request, openshift):
     Return an instance of OpenShift metrics client
     Skip tests if query route is not properly configured
     """
+    openshift_monitoring = openshift.change_project("openshift-monitoring")
+    # Check if metrics are enabled
+    try:
+        with openshift_monitoring.context:
+            cm = selector("cm/cluster-monitoring-config").object(cls=ConfigMap)
+            assert yaml.safe_load(cm["config.yaml"])["enableUserWorkload"]
+    except:  # pylint: disable=bare-except
```

Comment:
```suggestion
    except Exception:
```

Reply:
That would cause `too-broad-except` instead